docs/jade-getting-started.md

```diff
@@ -262,6 +265,21 @@ create extension pgcrypto;
   * Google Project Name: `broad-jade-ZZ` (replacing `ZZ` with your initials)
   * Google Organization: broadinstitute.org
 
+6. `ctrl-d` out of the psql shell and run the following to connect to your personal instance database:
+
+```
+cd jade-data-repo/ops
+DB=datarepo-ZZ SUFFIX=ZZ ENVIRONMENT=dev ./db-connect.sh
+```
+
+When in the psql shell again, run:
+
+```
+insert into billing_profile
+values ('390e7a85-d47f-4531-b612-165fc977d3bd', 'default', 'direct', '<BILLING_PROFILE>', null, '11/6/2020 8:29:59 AM', null);
```

Comment:
Can you add a note to set DROPALL_ON_START to false in datarepo-helm-definitions/dev/XX? Otherwise we will need to keep adding this back in.

Reply:
Preface: I was the one that came up with the solution of adding this db entry, but I'm not sure it's the best way.  It might be worth chatting about this addition as team.
 
I think our norm is to set dropAllOnStart = true for our personal dev instance. This keeps bad test runs from getting the db in a bad state. 

Proposed alternate solution: Instead of running this manual db entry, you can just run an integration test that create a billing profile. I don't like the manual db entry because it relies on there being a matching entry in the dev database, and if you do need to clear out the database, you'll have to re-add this multiple times.

Reply:
Got it! Is there a particular test we can point to in these docs?

Reply:
Something like the SnapshotTest.snapshotRowIdsHappyPathTest() might be good - it creates a billing profile, creates a dataset, does an ingest and then creates a snapshot.